CONTRIBUTING.md

```diff
@@ -174,7 +174,7 @@ For example:
 
 - to run a single test case for in-process FerretDB with `pg` handler
   with all subtests running sequentially,
-  you may use `env GOFLAGS='-run=TestName/TestCaseName -parallel=1' task test-integration-pg`;
+  you may use `env RUN='TestName/TestCaseName' task test-integration-pg`;
```

Comment:
The text above should be updated: it mentions only `GOFLAGS`, says "with all subtests running sequentially" (that is not true in that example), etc